core/src/main/java/module-info.java

```diff
@@ -577,6 +577,9 @@
 //                  avg()
             io.questdb.griffin.engine.functions.groupby.AvgDoubleGroupByFunctionFactory,
 //                  ^
```

Comment:
nit: this comment belongs to `io.questdb.griffin.engine.functions.math.PowDoubleFunctionFactory` and now it's shifted.